### DISCUSSION TITLE
If model can receive ibl light in a cube ?

### DISCUSSION THREAD

**Discussion by zhenhuafan** [google/filament#4467] [Q&A]
Because of the skybox don't have real ground, I built  a cube to simulate the skybox.
But the model in the cube can't show because of no any light can reach the inside of the cube.
If there is some method make the light can reach the inside for the cube ? For example some special material for the cube ?

Thanks for any suggestion!

**Comment by romainguy** [comment]
The IBL isn't shadowed so things should be lit inside the cube.

**Reply by zhenhuafan** [reply]
If the material of the cube can affect the model inside ?
